config-templates/ibm/ibm-csi-block/1.4.0/storage-class-template.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: "{{ sc-name }}"
```

Comment:
Change "sc-name" to "name"
and add add new variable for is-default-class

. Please refer https://github.com/IBM/ibm-satellite-storage/blob/develop/config-templates/netapp/netapp-ontap-san/20.07/storage-class-template.yaml

Reply:
done